setup.py

```diff
@@ -29,7 +29,10 @@ def main():
                 "nose",
                 "requests",
                 "xmlschema",
-            ]
+            ],
+            "mypy": [
+                "mypy==v0.740",  # keep this in sync with .pre-commit-config.yaml.
```

Comment:
What's the advantage of adding an extra over specifying a `deps` in ` [testenv:mypy]` tox?

Reply:
If e.g. pre-commit could also use the extra, the version could be changed just there.
Given requirements.txt, tox.ini etc I often find it more sane to have this in a more central place like setup.py.

Reply:
OK, makes sense.

I fear though that people will see it and start doing `pytest[mypy]` due to a misconception. There are probably tools which parse the extras and show them to the user. So WDYT about changing the name to `mypy-internal` or `_mypy` or something like that?

Reply:
Good point.  What about `checkqa-mypy`?

Reply:
That's clear enough I think.

Reply:
Great, amended.